data_access.go

```diff
@@ -605,6 +673,7 @@ func (s *DataAccessSyncer) exportDataAccess(config *data_access.DataAccessSyncCo
 
 		err = mergeGrants(conn, rn, foundGrants, expectedGrants)
 		if err != nil {
+			logger.Error("Encountered error :" + err.Error())
```

Comment:
On multiple locations in the code:
This is a bit of a weird error message. Either add extra context to the error or just only log the error.